src/components/NpsSurvey.js

```diff
@@ -137,18 +137,17 @@ export const NpsSurvey = Utils.connectAtom(authStore, 'authState')(class NpsSurv
           }, 'Submit')
         ])
       ]),
-      h(Clickable, {
-        as: icon('times-circle', {
+      h(Clickable, { onClick: goAway(false) }, [
+        icon('times-circle', {
```

Comment:
Why this change?

Reply:
The old library would accept an element instead of a component for `as`, but we don't.

Reply:
Indeed, I meant why _that_ change

Reply:
Accepting elements would require special detection and handling for that case, which increases the API surface area and the complexity of the component. I think this is a good simplification.